source/index.html.md

```diff
@@ -7382,10 +7383,10 @@ A transaction (debit or credit) can have the following statuses:
 
 | Status | Description |
 |--------|-------------|
-| `maturing` | The maturation date has not yet been reached. |
-| `matured` | The maturation date has been reached and the transaction is eligible for processing. |
 | `preprocessing` | The transaction is undergoing pre-checks before being sent to the bank. |
 | `processing` | The transaction has been submitted to the bank. |
+| `maturing` | The maturation date has not yet been reached. |
+| `matured` | The maturation date has been reached and the transaction is eligible for processing. |
```

Comment:
@corroded Should these stay in the order that they were? My thoughts are that this may cause some confusion for merchants as it reads as if preprocessing and processing happen before maturing and matured.

Reply:
That's my bad! Ill fix that after this release!

Reply:
I can fix this easy